src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/BackgroundDocumentGenerator.cs

```diff
@@ -213,10 +211,15 @@ private async Task Timer_TickAsync(CancellationToken cancellationToken)
 
                 lock (_work)
                 {
+                    // Suppress analyzer that suggests using DisposeAsync().
```

Comment:
📝 I had to do this in a few places. `Timer` implements `IAsyncDisposable` on .NET 7 and the VS Threading analyzers add a warning that we should be Calling `DisposeAsync`.